### ISSUE TITLE
Model training

### ISSUE THREAD

**Issue by Ljs1849351839** [LiLabAtVT/DeepTE#44]
Hi,

I tried to train DeepTE using my own TE dataset by following the same training procedure as in training_example_dir.

According to your previous replies to other users, I modified the following files:

In training_example_dir/classify_TE_keras_model_predict_kmer.py,
I changed the class_num under All to match my number of TE classes.

In training_example_dir/one_hot_rep_kmer.py,
I also changed the class definitions under All to my own categories.

However, when I ran the training, the model got stuck at around 0.3411 accuracy, and the validation accuracy did not improve throughout the training.
Here is the output:
Epoch 49/50
216375/216375 [==============================] - 245s 1ms/step - loss: 1.9938 - acc: 0.3411 - val_loss: 1.9936 - val_acc: 0.3411
Epoch 50/50
216375/216375 [==============================] - 243s 1ms/step - loss: 1.9938 - acc: 0.3411 - val_loss: 1.9936 - val_acc: 0.3411
54094/54094 [==============================] - 12s 229us/step

And the classification report is as follows:
              precision    recall  f1-score   support

     Class 0       0.34      1.00      0.51     18452
     Class 1       0.00      0.00      0.00     12166
     Class 2       0.00      0.00      0.00      5067
     Class 3       0.00      0.00      0.00      4093
     Class 4       0.00      0.00      0.00      3924
     Class 5       0.00      0.00      0.00      3393
     Class 6       0.00      0.00      0.00      2124
     Class 7       0.00      0.00      0.00      1735
     Class 8       0.00      0.00      0.00       892
     Class 9       0.00      0.00      0.00       706
    Class 10       0.00      0.00      0.00       456
    Class 11       0.00      0.00      0.00       437
    Class 12       0.00      0.00      0.00       260
    Class 13       0.00      0.00      0.00       151
    Class 14       0.00      0.00      0.00       120
    Class 15       0.00      0.00      0.00       118

    accuracy                           0.34     54094
   macro avg       0.02      0.06      0.03     54094
weighted avg       0.12      0.34      0.17     54094

It seems the model predicts everything as Class 0.

I’d like to ask:

Is this behavior normal when adapting DeepTE to new datasets?

Could it be caused by how I modified the class numbers or class definitions?

Do I need to make changes elsewhere (e.g., label encoding, data loading, or class mapping)?

Thank you very much for your time and for sharing DeepTE!

**Comment by Ljs1849351839** [comment]
非常感谢你的回复。非常感谢您在百忙之中抽出时间和精力。
